src/nvim/ex_docmd.c

```diff
@@ -4519,13 +4519,14 @@ static linenr_T get_address(exarg_T *eap, char **ptr, cmd_addr_T addr_type, int
           (void)hasFolding(lnum, NULL, &lnum);
         }
         if (i == '-') {
-          lnum -= n;
+          assert(INT32_MIN <= n && n <= INT32_MAX);
```

Comment:
This triggers the assert: `:-3333333333print`

Reply:
I don't understand what you mean.

Reply:
Can't you just change `n` to `linenr_T`?

Reply:
> I don't understand what you mean.

Build and run Nvim, type `:-3333333333print` and press, and you'll get a SIGBART.